JWBestPracticeApps/JWCasting/README.md

```diff
@@ -4,13 +4,16 @@ To enable casting to Google Cast with the JW Player iOS SDK, you must import the
 
 The JWPlayerController API controls the playback of the video being casted, and the JWPlayerDelegate will provide you with the playback callbacks while casting.
 
-Please note that Google Cast support is still in beta. The JW Player SDK supports casting to the Default Media Receiver and to Styled Media Receivers. Custom Receivers are not yet officially supported, but may work if the video playback implements the same interface as the Default Media Receiver. To specify a receiver, set the receiver's app ID to the `chromeCastReceiverAppID` property of the JWCastController.
+Please note that the JW Player SDK supports casting to the Default Media Receiver and to Styled Media Receivers. Custom Receivers are not yet officially supported, but may work if the video playback implements the same interface as the Default Media Receiver. To specify a receiver, set the receiver's app ID to the `chromeCastReceiverAppID` property of the JWCastController.
 
-# Known issues with Google Cast:
+# The following features are not supported during a casting session with an iOS SDK player:
 
-* Google IMA ads are not supported when casting.
-* Multiple AudioTracks or AudioTrack switching is not supported when casting.
-* Only WebVTT captions are supported; support for other caption formats will be added in the future
+* Google IMA ads
+* FreeWheel ads
+* 608 captions
+* DVR and live streaming capabilities
+* Multiple-audio tracks or AudioTrack switching (not supported natively by Chromecast)
```

Comment:
@askolodny  out of curiosity, what do we mean by natively ? Do we mean not supported out of the box ? Or is native here used in the context of native SDK

Reply:
@karimMourra I based it off of the language which I thought you worked on with Kevin for the developer documentation: 

"Chromecast does not support these features natively."

See: https://developer.jwplayer.com/sdk/ios/docs/developer-guide/casting/chromecast/

Do you think that it should be reworked? Happy to do reword it.

Reply:
We did go over these but i don't know how `natively` ended up there. My explanation was that these features are not supported out of the box with the default receiver. In order to have them, you have to implement these features in a custom receiver.

Reply:
@karimMourra I'm confused. This is the copy that was published:

> **FAQ**
> Which features not supported when casting with an iOS SDK player?
> 
> The following features are not supported during a casting session with an iOS SDK player:
> 
> - Google IMA ads
> - FreeWheel ads
> - Multiple-audio tracks or AudioTrack switching^1
> - In-manifest WebVTT captions^1
> - 608 captions
> - DVR and live streaming capabilities
> 
> ^1^ Chromecast does not support these features natively.

Based on that (and what I have been telling customers based off of this) is that multi-audio tracks or audiotrack switching and in-manifest webvtt captions are not supported by chromecast at all. BUT Google IMA and Freewheel ads, 608 captions, as well as DVR and live streaming are but will require custom receiver support.

Are you saying that is not the case?  FYI @kcorneli201 and @henryhlee

Reply:
yes. If you look at the custom receiver docs: 
https://developers.google.com/cast/docs/caf_receiver/tracks
The docs say it supports WebVTT, i assume it means in manifest, since side-loaded is already supported in the default receiver.